### PR TITLE
Add the game loop when a valid move is made and add alternating turns

### PR DESCRIPTION
**Pull request by jonmanahan**
Calculates the turn each iteration and loops, ending when a player
has won or the game has tied. Also only displays the welcome message
once.

### CODE REVIEW

File: lib/tic_tac_toe.ex [hunk 1]
```diff
@@ -5,26 +5,34 @@ defmodule TicTacToe do
...
   def start(communicator, communicator_formatter, board \\ Board.setup_initial_board()) do
     welcome = @welcome_message <> "\n" <> communicator_formatter.format_board(board)
     communicator.display(welcome)
 
+    play_turn(communicator, communicator_formatter, board)
+  end
+
+  @spec play_turn(any(), any(), map()) :: nil
+  defp play_turn(communicator, communicator_formatter, board) do
+    player_turn = Board.calculate_turn(board)
```
Comment (elijahsullivan) [inline lib/tic_tac_toe.ex:18]: I'm not sure what I would expect a name like `player_turn` to represent, but I don't think of the markings/symbols of "X" and "O" being "player turns" personally. I wonder if we could come up with another name to represent these.

Comment (jonmanahan) [reply lib/tic_tac_toe.ex:18]: That is a really good point, `player_symbol` sounds much better to me!

**Review by elijahsullivan — APPROVED** [review]
LGTM

**Comment by jonmanahan** [comment]
> LGTM

Awesome! Thank you for the review and help Eli! 😄

**Pull request merged**